lib/Fmt_ast.ml

```diff
@@ -369,15 +369,17 @@ let fmt_direction_flag = function
 let fmt_virtual_flag f =
   match f with Virtual -> fmt "@ virtual" | Concrete -> noop
 
+type parsed_docstring = Verbatim | Parsed of Odoc_parser.Ast.docs
+
 let parse_docstring ~loc text =
   let location = loc.Location.loc_start in
   let location =
     { location with
       pos_cnum= location.pos_cnum + 3 (* Length of comment opening *) }
   in
-  let parsed = Odoc_parser.parse_comment_raw ~location ~text in
-  match parsed with
-  | {value; warnings= []} -> Ok value
+  match Odoc_parser.parse_comment_raw ~location ~text with
+  | exception _ -> Ok Verbatim
```

Comment:
I think we should return an `Error` here.
What's wrong with printing "Warning: Invalid documentation comment" ?

The error message will change depending on the current Odoc version but we already expect that, each versions of Odoc could change error messages and error conditions.

An unstable test is an issue but we don't need to make OCamlformat's stderr output always stable (it's not).

Reply:
I tend to agree with you, @emillon what do you think?

Reply:
Yes, that seems a good point to me as well. To clarify, I didn't mean to imply that stderr should be stable, just that it can't be tested in a stable way.

Reply:
done